devolving_music/lib/score_object.py

```diff
@@ -35,8 +35,10 @@ def compare_present(self, comparison: "SongComparison"):
         return comparison.id in self.counted_compares
 
     def __str__(self):
-
-        string_output = f"Song:{self.song_submission} \n Energy Score:{self.energy_score}\n Quality Score:{self.quality_score} \n Post Peak Score:{self.post_peak_score}\n Info Score:{self.info_score} "
+        song_sub=SongSubmission.objects.get(id__exact=self.song_submission)
+        song=SongSubmission.objects.get(id__exact=self.song_submission).song
+        artist_list=list(musician.name for musician in song.artists.all())
+        string_output = f" Song Submission ID:{song_sub.id} \n Song:{song.title} by {artist_list} \n Energy Score:{self.energy_score}\n Quality Score:{self.quality_score} \n Post Peak Score:{self.post_peak_score}\n Info Score:{self.info_score} "
```

Comment:
This was really only intended for quickly displaying scoring information for testing end users won't see it because end users never have to see actual score data and even if we do want them to that'd probably be handled in a new view

Reply:
but I fixed the colon and space thing

Reply:
developers are people too 😛